foundry.toml

```diff
@@ -1,6 +1,7 @@
 [profile.default]
   optimizer = true
   optimizer_runs = 10_000_000
+  remappings = ["@openzeppelin=lib/flexible-voting/lib/openzeppelin-contracts/"]
```

Comment:
repo wouldn't build with the latest version of foundry until I added this

Reply:
Yes, I saw this with a bunch of my repos as well

Reply:
for reference I think this is related to https://github.com/foundry-rs/foundry/issues/5475